test_flake.py

```diff
@@ -0,0 +1,3 @@
+import nimp
```

Comment:
we don't need this file do we?

Reply:
That's right, thanks, I forgot I created it in `metric-learn`, and added it by mistake...